ionoscloud/resource_networkloadbalancer.go

```diff
@@ -69,8 +71,19 @@ func resourceNetworkLoadBalancer() *schema.Resource {
 				ForceNew:         true,
 				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotWhiteSpace),
 			},
+			"flowlog": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     cloudapiflowlog.FlowlogSchemaResource,
+				MaxItems: 1,
+				Description: `Only 1 flow log can be configured. Only the name field can change as part of an update. Flow logs holistically capture network information such as source and destination 
```

Comment:
@cristiGuranIonos why is this indentation different than the one in the data source?